app/main.py

```diff
@@ -1,3 +1,57 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, distance: int) -> None:
+        self.km = distance
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int) -> Distance:
+        if type(other) == Distance:
```

Comment:
Although there are no errors in logic, use `isinstance` over `type` for this case and others, you can read the reasoning for that [here](https://blog.devgenius.io/what-is-better-type-vs-isinstance-3b948d1b81b5) and [here](https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance).